SPECS/libcontainers-common/libcontainers-common.spec

```
@@ -16,35 +16,35 @@
 #
 
 # commonver - version from containers/common
-%define commonver 0.14.6
+%define commonver 0.44.0
 # podman - version from containers/podman
-%define podmanver 2.0.3
+%define podmanver 3.3.1
 # storagever - version from containers/storage
-%define storagever 1.20.2
+%define storagever 1.36.0
 # imagever - version from containers/image
-%define imagever 5.5.1
+%define imagever 5.16.0
 Summary:        Configuration files common to github.com/containers
 Name:           libcontainers-common
-Version:        20200727
-Release:        2%{?dist}
-License:        ASL 2.0 AND GPLv3+
+Version:        20210626
+Release:        1%{?dist}
+License:        Apache-2.0 AND GPL-3.0-or-later
```

Please use the [short names similar to what Fedora does](https://fedoraproject.org/wiki/Licensing:Main?rd=Licensing#Good_Licenses).

```suggestion
License:        ASL 2.0 AND GPLv3+
```

---

updated.